test.py

```diff
@@ -835,31 +839,25 @@ def write_junit_failure_report(self, xml_res: ET.Element) -> None:
 
 class TopologyTest(PythonTest):
     """Run a pytest collection of cases against Scylla clusters handling topology changes"""
-    is_before_test_ok: bool
-    is_after_test_ok: bool
     status: bool
 
     def __init__(self, test_no: int, shortname: str, suite) -> None:
         super().__init__(test_no, shortname, suite)
 
     async def run(self, options: argparse.Namespace) -> Test:
 
-        async with get_cluster_manager(self.shortname, self.suite.clusters) as manager:
-            self.args.insert(0, "--host={}".format(manager.cluster[0].host))
-            logging.info("Leasing Scylla cluster %s for test %s", manager.cluster, self.uname)
+        test_path = os.path.join(self.suite.options.tmpdir, self.mode)
+        async with get_cluster_manager(self.shortname, self.suite.clusters, test_path) as manager:
+            self.args.insert(0, "--manager-api={}".format(manager.sock_path))
 
             try:
-                manager.cluster.before_test(self.uname)
-                self.is_before_test_ok = True
                 manager.cluster[0].take_log_savepoint()
                 status = await run_test(self, options)
-                manager.cluster.after_test(self.uname)
-                self.is_after_test_ok = True
                 self.success = status
             except Exception as e:
                 self.server_log = manager.cluster[0].read_log()
                 self.server_log_filename = manager.cluster[0].log_filename
-                if self.is_before_test_ok is False:
+                if manager.is_before_test_ok is False:
```

Comment:
Where is `manager.is_before_test_ok` set to `True`?

Reply:
Missed on refactor. I'll fix.

Reply:
Fixed in latest.